app.js

```diff
@@ -1,11 +1,61 @@
-var express = require('express');
-var app = express();
+// Load libraries
+var express = require('express')
+  , app = express()
+  , mongoose = require('mongoose')
+  , bodyParser = require('body-parser')
+  , morgan = require('morgan')
+  , cookieParser = require('cookie-parser')
+  , expressSession = require('express-session')
+  , csrf = require('csurf')
+  , handlebars  = require('express-handlebars');
 
-app.get('/', function(req, res) {
-  res.send('Hello, world!');
+// Load configuration and routes
+var port = process.env.PORT || 3000 // port 3000 as default
+  , passport = require('./config/passport')
+  , DB = require('./config/database')
+  , authRoutes = require('./routes/auth');
+
+// set up express middlewares
+app.use(morgan('dev')); // log every request to the console
+app.use(cookieParser()); // read cookies (needed for csurf)
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+// Initiate Passport.js for authentication
+app.use(expressSession({
+  secret: 'testexpresssession'
+  , resave: true
+  , saveUninitialized: false
+  , cookie: {
+    httpOnly: true
+    , secure: false // this can be set only if HTTPS
+  }
+}));
```

Comment:
It's easy to miss a comma at the end of a line.

Another advantage of the style is when you want to add a new line after the last line, you just add it, without putting a comma in end of last line. In diff system it looks cleaner too.